external/dhcpd/dhcpd_lwip.c

```diff
@@ -209,6 +209,9 @@ int dhcp_server_status(char *intf)
 
 int dhcp_server_start(char *intf, dhcp_sta_joined dhcp_join_cb)
 {
+	pthread_attr_t attr;
+	struct sched_param sparam;
+	int ret= ERROR;
```

Comment:
Some of defect detect tool will treat it as a minor issue because of line 218 & 224, it never used.
So you had better return ret, instead of return ERROR.